tests/ui_tools/test_popups.py

```diff
@@ -179,6 +180,44 @@ def test_keypress_navigation(
 
         self.super_keypress.assert_called_once_with(size, navigation_key)
 
+    @pytest.mark.parametrize(
+        [
+            "contents",
+            "column_widths",
+            "expected_instance",
```

Comment:
`instance` is rather vague - what does this function return?

I believe each part of the `instance` is essentially a row, which we don't mention in the docstring and maybe in the code that uses it, but could be a useful refactor?

Reply:
Changed the parameter name to `expected_widget_type` to be more descriptive since the docstring also mentions that the function is returning list of widgets.

Re second part, are you saying each widget of the list is essentially a row and that the docstring needs a refactor? I feel the current docstring is good enough, it does return a list of widgets that is used to make the table.

Reply:
Using `widget` is consistent with the current docstring and terms used :+1:

The second point was my noting that the code might be more readable if we referred to the returned data as rows; widgets are any UI element, so are less specific.